packages/sdk/contractkit/src/celo-tokens.ts

```diff
@@ -68,7 +59,7 @@ export const celoTokenInfos: {
  * A helper class to interact with all Celo tokens, ie CELO and stable tokens
  */
 export class CeloTokens {
-  constructor(readonly kit: ContractKit) {}
+  constructor(readonly contracts: ContractCacheType, readonly registry: AddressRegistry) {}
```

Comment:
I think that would be of limited use. In the past this required a ContractKit Instance, but creating an instance of ContractKit itself makes an instance of CeloTokens so in the past it would have been very odd to initialize this directly

Reply:
Makes sense!